test/sql/optimizer_sql_test.cpp

```diff
@@ -165,12 +170,12 @@ TEST_F(OptimizerSQLTests, SimpleSelectTest) {
 
   // Testing predicate
   TestUtil("SELECT c, b from test where a=1", {"333", "22"}, false);
-  
+
   // Something wrong with column property.
   query = "SELECT a, b, c from test order by a + c";
 
   // check for plan node type
-  select_plan = TestingSQLUtil::GeneratePlanWithOptimizer(optimizer, query);
+//  select_plan = TestingSQLUtil::GeneratePlanWithOptimizer(optimizer, query);
```

Comment:
The variable 'select_plan' has never been used in the following part. '//  EXPECT_EQ(select_plan->GetPlanNodeType(), PlanNodeType::ORDERBY);' this test is also commented out by others.